angio-frontend/src/modules/organization/components/OrganizationListTable.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+    <v-layout row wrap>
+        <v-flex xs12>
+            <v-text-field
+                    v-bind:label="$t('organization.component.organizationListTable.table.searchField')"
+                    v-model="searchField"
+                    append-icon="mdi-magnify"
+                    type="search"
+                    browser-autocomplete="off"
+                    clearable
+                    hide-details
+                    solo
+            ></v-text-field>
+        </v-flex>
+
+        <v-flex xs12>
+            <v-card>
+                <v-data-table
+                        v-bind:headers="headers"
+                        v-bind:items="organizations"
+                        v-bind:loading="fetching"
+                        v-bind:pagination.sync="pagination"
+                        v-bind:total-items="totalItems"
+                        v-bind:no-data-text="$t('organization.component.organizationListTable.table.noDataText')"
+                        item-key="id"
+                        sort-icon="arrow_drop_down"
+                        disable-initial-sort
+                        hide-actions
+                >
+                    <template slot="headerCell" slot-scope="props">
+                        {{ $t(props.header.text) }}
+                    </template>
+                    <template v-slot:items="props">
+                        <td>{{ props.item.id }}</td>
```

Comment:
Add search result highlighting. Check `UserListTable` and `AnalyseListTable`
`
<text-highlight v-bind:queries="searchForHighlight">
       {{ props.item.id }}
</text-highlight>
`